src/library/trackset/setlogfeature.cpp

```diff
@@ -394,28 +451,28 @@ void SetlogFeature::slotPlaylistTableRenamed(int playlistId, const QString& newN
     }
 }
 
+void SetlogFeature::activate() {
+    activatePlaylist(m_playlistId);
+}
+
+void SetlogFeature::activatePlaylist(int playlistId) {
+    //qDebug() << "BasePlaylistFeature::activatePlaylist()" << playlistId;
```

Comment:
In this function you don't check m_pSidebarWidget while in BasePlaylistFeature you do.

Just noticed this inconsistency, not sure how to resolve it.

Reply:
activateChild checks if m_pSidebarWidget is set and selects the proper index. The checks are only in startup of relevance.